doc/Caliptra.md

```diff
@@ -1134,131 +1134,130 @@ Notes:
 
 ## Boot Media Integrated
 
-### Caliptra FW Load Flow
+### Caliptra FW load flow
 
-*Figure 12: FW Load Flow*
+*Figure 12: FW load flow*
 
 ![](./images/Caliptra_boot_flow4.png)
 
-1. Once Caliptra uController is out of reset, ROM starts executing and triggers the crypto block to run the UDS decrypt flow.
-2. Caliptra ROM will use the internal SPI peripheral to read from the platform's persistent storage and load the FW. Note that SPI will be operating in basic functional single-IO mode and at 20 MHz frequency.
-3. Once Caliptra’s FW is authenticated and loaded into ICCM, uController will assert READY\_FOR\_RTFLOWS wire. Refer to [Caliptra ROM specification](https://github.com/chipsalliance/caliptra-sw/blob/main/rom/dev/README.md) for next level details of various security flows that happen within this step (eg. DICE).
+1. After the Caliptra uController is out of reset, ROM starts executing and triggers the crypto block to run the UDS decrypt flow.
+2. Caliptra ROM uses the internal SPI peripheral to read from the platform's persistent storage and load the FW. Note that SPI is operating in basic functional single-IO mode and at 20 MHz frequency.
+3. After Caliptra’s FW is authenticated and loaded into ICCM, uController asserts READY\_FOR\_RTFLOWS wire. See the [Caliptra ROM specification](https://github.com/chipsalliance/caliptra-sw/blob/main/rom/dev/README.md) for specific information about security flows that happen within this step (for example, DICE).
 
-### <a id="reset-flow"></a>CPU Warm Reset or PCIe Hot Reset Flow →  Caliptra IP reset
+### <a id="reset-flow"></a>CPU warm reset or PCIe hot reset flow →  Caliptra IP reset
 
-*Figure 13: Hardware Reset Flow*
+*Figure 13: Hardware reset flow*
 
 ![](./images/Caliptra_boot_flow5.png)
 
-**Note:** Since Caliptra IP may be placed in a “S5” domain of the device, there may be devices where Caliptra IP may not go through reset on a device hot reset or CPU warm reset. But the flow shows what happens when such a reset happens.
+**Note:** Since Caliptra IP may be placed in an “S5” domain of the device, there may be devices where Caliptra IP may not go through reset on a device hot reset or CPU warm reset. But the flow shows what happens when such a reset happens.
```

Comment:
`S5` should be qualified or defined or something.

Reply:
created issue